src/Components/App/App.js

```diff
@@ -31,25 +31,31 @@ class App extends Component {
     }
   }
 
-  getActivityData = async (event) => {
+  getActivityData = async (name) => {
+    if (name.target) {
+      name = name.target.id
+    }
+    if (name.includes('games')) {
+      name = name.replace(/ games/gi, 'Games')
+    }
     let promise;
     try {
-      if (event.target.id === 'movies') {
+      if (name === 'movies') {
         promise = await getAllMovies();
       }
-      if (event.target.id === 'boardGames') {
+      if (name === 'boardGames') {
         promise = await getAllBoardGames();
       }
-      if (event.target.id === 'cardGames') {
+      if (name === 'cardGames') {
         promise = await getAllCardGames();
       }
-      if (event.target.id === 'music') {
+      if (name === 'music') {
         promise = await getAllMusic();
       }
-      if (event.target.id === 'podcasts') {
+      if (name === 'podcasts') {
         promise = await getAllPodcasts();
       }
-      this.setState({[event.target.id]: promise})
+      this.setState({[name]: promise})
     } catch (error) {
       console.log(error)
```

Comment:
I think we want to assign the error to a property in the state so we can render appropriate content, such as failed to retrieve for error handling.